src/docs/release/breaking-changes/index.md

```diff
@@ -11,8 +11,10 @@ currently available:
 * [TextInputClient currentTextEditingValue]
 * [TestTextInput]
 * [Scrollable AlertDialog]
+* [Generic type of ParentDataWidget changed to ParentData]
```

Comment:
Could you move this up, so that it's an alphabetical list? I think. :)

Reply:
I did move this up. Although, I am wondering if this list should be in "chronological" order instead. When I look at the list, I am probably only interested in the latest guides and don't want to scroll through the guides that are 25 versions old?